README.md

```diff
@@ -3,53 +3,47 @@
 You will need to use 'create-testset' action once and 'populate-testset' action every time there is a new report.
 If the structure of the report changes (new tests added for example), you will need to use 'create-testset' action again. New TestSet will be created, but existing tests will be reused.
 
+If you don't have exsiting CONFIG_FILE and you want to use it go to https://firecracker-ui-prod.practitest.com/ and generate one.
+To login you can use your PractiTest credentials, follow the instructions in the link and then you can continue here, set config-path to your configuration file path.
 ## Usage
 
+### help
+
+At evrey point you can run:
+
+``` shell
+java -jar practitest-firecracker-standalone.jar help
+```
+to get more information about the parameters and commands.
+
 ### create-testset
 
 Create new testset from reports folder:
 
 ``` shell
-java -jar practitest-firecracker-standaline.jar \
-    --api-token=YOUR_API_TOKEN \
-    --email=YOUR_EMAIL \
+java -jar practitest-firecracker-standalone.jar \
     --reports-path=SUREFIRE_REPORTS_PATH \
-    --project-id=PRACTITEST_PROJECT_ID \
     --testset-name="TestSet name" \
     --author-id=PRACTITEST_USER_ID \
+    --config-path=CONFIG_FILE \
     create-testset
 ```
 
 The call above will analyze the surefire reports and create tests and testset. If a test already exists, it will be reused.
-You can set various custom fields for tests when they are created (especially useful if you have mandatory fields configured in your tests).
-Here is an example for setting custom fields during creation of the tests and testset:
 
-``` shell
-java -jar practitest-firecracker-standaline.jar \
-    --api-token=YOUR_API_TOKEN \
-    --email=YOUR_EMAIL \
-    --reports-path=SUREFIRE_REPORTS_PATH \
-    --project-id=PRACTITEST_PROJECT_ID \
-    --testset-name="TestSet name" \
-    --author-id=PRACTITEST_USER_ID \
-    --additional-test-fields '{"custom-fields": {"---f-123": "foo", "---f-124": "bar"}}' \
-    --additional-testset-fields '{"custom-fields": {"---f-125": "baz"}}' \
-    create-testset
-
-```
+You can set various custom fields for tests when they are created (especially useful if you have mandatory fields configured in your tests).
 
-Replace the field ids with the actual IDs, you can see all your custom fields by calling this API call: [get-all-custom-fields-in-your-project](https://www.practitest.com/api-v2/#get-all-custom-fields-in-your-project)
+Those configuration are set in config file to edit the config file go to https://firecracker-ui-prod.practitest.com/ and then use the example above again with the new generated config file.
```

Comment:
Don't think it's valid english.

Reply:
Sorry, I got used to Grammarly to auto-correct me, will rewrite.